plutus-core/cost-model/create-cost-model/BuiltinMemoryModels.hs

```diff
@@ -176,5 +173,8 @@ builtinMemoryModels = BuiltinCostModelBase
   -- paramCaseList
   -- paramCaseData
   , paramDropList                        = Id $ ModelTwoArgumentsConstantCost 4
+  , paramLengthOfArray                   = Id $ ModelOneArgumentConstantCost    99
```

Comment:
I'm trying to keep everything in historical order here.  If stuff gets added in the middle of functions or lists it's easy to miss that it's happened.